### ISSUE TITLE
pvacseq install_vep_plugin fail due to incorrect copy path

### ISSUE THREAD

**Issue by kylec** [griffithlab/pVACtools#75]
I created a python3.6 env using conda and did pip install pvactoolsas instructed. I had trouble installing the VEP_plugins and I realized that base_dir variable in install_vep_plugin.py already contains "/tools" in the path so wildtype_plugin_path ends up with double "/tool" at line 16
```
wildtype_plugin_path = os.path.join(base_dir, 'tools', 'pvacseq', 'VEP_plugins', 'Wildtype.pm')
```

Hence the following error message when I install vep_plugins. 
```
pvacseq install_vep_plugin VEP_plugins
Traceback (most recent call last):
  File "/rsrch2/ccp_rsch/kchang3/.conda/envs/pvactools-env/bin/pvacseq", line 11, in <module>
    sys.exit(main())
  File "/rsrch2/ccp_rsch/kchang3/.conda/envs/pvactools-env/lib/python3.6/site-packages/tools/pvacseq/main.py", line 78, in main
    args[0].func.main(args[1])
  File "/rsrch2/ccp_rsch/kchang3/.conda/envs/pvactools-env/lib/python3.6/site-packages/tools/pvacseq/install_vep_plugin.py", line 17, in main
    copyfile(wildtype_plugin_path, os.path.join(args.vep_plugins_path, 'Wildtype.pm'))
  File "/rsrch2/ccp_rsch/kchang3/.conda/envs/pvactools-env/lib/python3.6/shutil.py", line 120, in copyfile
    with open(src, 'rb') as fsrc:
FileNotFoundError: [Errno 2] No such file or directory: '/rsrch2/ccp_rsch/kchang3/.conda/envs/pvactools-env/lib/python3.6/site-packages/tools/tools/pvacseq/VEP_plugins/Wildtype.pm
```

**Comment by kylec** [comment]
I manually created VEP_plugins in the pvacseq directory and made two changes in install_vep_plugin.py and it works now.

base_dir = os.path.abspath(os.path.join(os.path.dirname(os.path.realpath(__file__)), '..'))
    #wildtype_plugin_path = os.path.join(base_dir, 'tools', 'pvacseq', 'VEP_plugins', 'Wildtype.pm')
    wildtype_plugin_path = os.path.join(base_dir, 'pvacseq', 'VEP_plugins', 'Wildtype.pm')
    #copyfile(wildtype_plugin_path, os.path.join(args.vep_plugins_path, 'Wildtype.pm'))
    copyfile(os.path.join(args.vep_plugins_path, 'Wildtype.pm'), wildtype_plugin_path)

**Comment by kylec** [comment]
i realize the main issue was "pip install pvactools" doesn't generate iedb_alleles and VEP_plugins directories, unlike standalone pvacseq install. 

Is pvacseq (4.0.10) the same as the tools/pvacseq (1.0.0)?

**Comment by susannasiebert** [comment]
Thank you for this bug report and for digging into this problem. I apologize for the errors you are running into. 

pVACseq as part of pVACtools 1.0.0 is based on `pvacseq 4.0.10` but we added some new features and moved a lot of the code base around. I think this is a simple problem with the setup script pointing to an outdated path. I will do some digging and report back soon.

**Comment by susannasiebert** [comment]
This issue has been resolved in version 1.0.1. Please update your local pvactools install to the newest version.
